### PR TITLE
Remove getfinishstatus and processFace

### PR DESCRIPTION
**Pull request by levhitaIncode**
* Updated dependencies
* moved calls to backend to another file like other examples
*removed processFace
* replaces getFinishStatus with call to backend
* unified method names with other examples

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,28 +1,13 @@
 
+import { startOnboardingSession, finishOnboardingSession } from './session'
+
 // Lets put all the variables needed for all modules in the global scope
-const tokenServerURL= import.meta.env.VITE_TOKEN_SERVER_URL;
```
Comment (levhitaIncode) [inline main.js:3]: encapsulated in session

File: main.js [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
 const localServerUrl= import.meta.env.VITE_LOCAL_SERVER_URL;
 
 let incode;
 let session;
 const container = document.getElementById("camera-container");
 
-async function startOnboardingSession() {
```
Comment (levhitaIncode) [inline main.js:10]: moved to session

File: main.js [hunk 3]
```diff
@@ -32,40 +17,41 @@ function renderRedirectToMobile(){
   if (incode.isDesktop()) {
     incode.renderRedirectToMobile(container, {
       onSuccess: () => {
-        finish();
+        showFinishScreen();
```
Comment (levhitaIncode) [inline main.js:20]: finish screen is not the same as finishing the onboarding, this clarifies that

File: main.js [hunk 4]
```diff
@@ -32,40 +17,41 @@ function renderRedirectToMobile(){
...
     });
   } else {
-    renderUserConsent();
+    saveDeviceData();
+    showUserConsent();
   }
 }
 
-function renderUserConsent(){
+function saveDeviceData() {
```
Comment (levhitaIncode) [inline main.js:32]: encapsulated devicedata

File: main.js [hunk 5]
```diff
@@ -32,40 +17,41 @@ function renderRedirectToMobile(){
...
-function renderFrontIDCamera() {
-  
-  // Optional but valuable for fraud prevention, hurts conversion
-  // incode.sendFingerprint(session);
-  // incode.sendGeolocation(session);
-  
+function captureIdFrontSide() {
   incode.renderCamera("front", container, {
-    onSuccess: renderBackIDCamera,
+    onSuccess: captureIdBackSide,
```
Comment (levhitaIncode) [inline main.js:46]: renamed to match all other js examples

File: main.js [hunk 6]
```diff
@@ -80,34 +66,27 @@ async function  processID() {
...
+  captureSelfie();
 }
 
-
-
-function renderSelfieCamera() {
+function captureSelfie() {
   incode.renderCamera("selfie", container, {
-    onSuccess: processFace,
+    onSuccess: finishOnboarding,
```
Comment (levhitaIncode) [inline main.js:74]: this one actually calls the backend

File: package.json [hunk 7]
```diff
@@ -9,7 +9,7 @@
     "preview": "vite preview"
   },
   "devDependencies": {
-    "vite": "^4.5.3",
+    "vite": "^5.2.13",
```
Comment (levhitaIncode) [inline package.json:12]: updated and tested

**Review by incodecota — APPROVED** [review]

**Pull request merged**